src/factory.service.js

```diff
@@ -26,7 +26,15 @@ const makeService = (cache, twitter, notifications) => {
             const reminderTime = mainResult.start.date();
             // Minimum of 3 minutes from now
             const minimumReminderInterval = (new Date(Date.now() + (3 * 60 * 1000)));
-            if (reminderTime >= minimumReminderInterval) {
+            // Maximum of 30 years from now
+            const maximumReminderInterval = (new Date((d => d.setFullYear(d.getFullYear() + 30))(new Date)));
```

Comment:
Lol, you're right; I do not like this. That's way too many brackets for me. How about:

```suggestion
            const maximumReminderInterval = new Date((new Date).setFullYear(d.getFullYear() + 30));
```

Reply:
Awesome, this is cleaner 👍🏾 will fix asap

Reply:
Hmm, do you mean this
```suggestion
            const maximumReminderInterval = new Date((new Date).setFullYear((new Date).getFullYear() + 30));
```